pykechain/models/scope2.py

```diff
@@ -86,16 +86,14 @@ def refresh(self, json=None, url=None, extra_params=None):
     # CRUD methods
     #
 
-    def _update_scope_project_team(self, select_action, user, user_type):
+    def _update_scope_project_team(self, select_action, user):
         """
         Update the Project Team of the Scope. Updates include addition or removing of managers or members.
 
         :param select_action: type of action to be applied
```

Comment:
what are the actions, defined in ENUM, please include in the docs as these are printed in the documentation of the code as well.